joomla/joomla-cms
# [4.1] Resolve Com_contact_email bug in default_address.php

Pull Request for Issue #36957 
### COM_CONTACT_EMAIL issue created by [dzandut]
76 <span class="visually-hidden"><?php echo Text::_('COM_CONTACT_EMAIL'); ?>"></span>




### Testing Instructions
Enable email display in contact form and then enter the any value and you will see the value

### Actual result BEFORE applying this Pull Request
Instead of value of variable , variable name will appear


### Expected result AFTER applying this Pull Request
Value of email will appear when you enable email display


### Documentation Changes Required
Not at all
